src/redux/selectors.js

```diff
@@ -1,3 +1,23 @@
+import { isWithinRange } from 'date-fns';
+import { find } from 'lodash';
+
+const getVelocity = (stories, sprint) =>
+  stories.reduce((result, story) => {
+    if (isWithinRange(story.closedAt, sprint.startsAt, sprint.endsAt)) {
+      return result + story.estimate;
+    }
+
+    return result;
+  }, 0);
+
+const getSprint = (sprints, story) => {
+  const storySprint = find(sprints, sprint =>
+    isWithinRange(story.closedAt, sprint.startsAt, sprint.endsAt),
+  );
+
+  return storySprint ? storySprint.number : 'None';
```

Comment:
🎨: Maybe `get(find(...), 'number', 'None')` here since we've got `lodash` around?